### PR TITLE
[14.0] [MIG] maintenance_equipment_hierarchy

### PR DESCRIPTION
**Pull request by mariadforgeflow**
Standard migration to v14

### CODE REVIEW

File: maintenance_equipment_hierarchy/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+{
+    "name": "Maintenance Equipments Hierarchy",
+    "summary": "Manage equipment hierarchy",
+    "author": "ForgeFlow, Odoo Community Association (OCA)",
+    "maintainers": ["dalonsod"],
+    "website": "https://github.com/OCA/maintenance",
+    "category": "Equipments, Assets, Internal Hardware, Allocation Tracking",
+    "version": "14.0.1.0.1",
```
Comment (dalonsod) [inline maintenance_equipment_hierarchy/__manifest__.py]: Wrong version:
```suggestion
    "version": "14.0.1.0.0",
```

File: maintenance_equipment_hierarchy/models/maintenance_equipment.py [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    _parent_name = "parent_id"
+    _parent_store = True
+    _parent_order = "name"
+
+    parent_id = fields.Many2one(
+        "maintenance.equipment",
+        "Parent Equipment",
+        index=True,
+        ondelete="cascade",
+        track_visibility="onchange",
```
Comment (dalonsod) [inline maintenance_equipment_hierarchy/models/maintenance_equipment.py]: This should be also changed in v13, as I've noticed:
```suggestion
        tracking=True,
```

**Review by dalonsod — CHANGES_REQUESTED** [review]
See comments. Anyway, I'm not sure why isn't passing runbot, could you check?

**Review by dalonsod — APPROVED** [review]
👍 Tested on runbot

**Review by flotho — APPROVED** [review]
tested on runbot,

LGTM

**Review by etobella — APPROVED** [review]

**Pull request merged**